src/App.css

```diff
@@ -19,24 +23,6 @@ ul {
   list-style: none;
 }
 
-.container {
```

Comment:
Interesting!

However, in my personal opinion I would like to say something:
Please 🙏 , take care with the rest app or pages because they might be broken. Maybe in this specific case, it's fine 👍 because there are other PR where those issues will be solved.

Welcome Page:

![image](https://user-images.githubusercontent.com/8689897/142732366-cb5fa5a3-bc64-45c6-a990-ff010c0969c4.png)

Maybe, it might be a team decision but you can create an extra PR, where you might put your "global"/app styles because the visualization of some pages is broken. Take care with it.